website/pages/en/querying/graphql-api.mdx

```diff
@@ -70,11 +81,12 @@ In the following example, we sort the tokens by the name of their owner:
 
 ### Pagination
 
-When querying a collection, the `first` parameter can be used to paginate from the beginning of the collection. It is worth noting that the default sort order is by ID in ascending alphanumeric order, not by creation time.
-
-Further, the `skip` parameter can be used to skip entities and paginate. e.g. `first:100` shows the first 100 entities and `first:100, skip:100` shows the next 100 entities.
+When querying a collection, it's best to:
 
-Queries should avoid using very large `skip` values since they generally perform poorly. For retrieving a large number of items, it is much better to page through entities based on an attribute as shown in the last example.
+- Use the `first` parameter to paginate from the beginning of the collection.
+  - The default sort order is by `ID` in ascending alphanumeric order, **not** by creation time.
```

Comment:
I would move this bullet point to the "Sorting" section above.

Reply:
Not sure if you disagree with this (as it's marked as resolved but it's still here), but just wanted to flag it again just in case. Sorting is only related to pagination in that it changes which results will appear on which page, but there's a section dedicated to Sorting on this page so it really feels like it belongs there.

Reply:
🤷‍♂️